nixos/lib/make-options-doc/default.nix

```diff
@@ -119,7 +119,11 @@ in rec {
       buildInputs = [
         pkgs.brotli
         (let
-          self = (pkgs.python3Minimal.override {
+          # python3Minimal can't be overridden with packages on Darwin, due to a missing framework.
+          # Instead of modifying stdenv, we take the easy way out, since most people on Darwin will
+          # just be hacking on the Nixpkgs manual (which also uses this mechanism).
```

Comment:
Would it maybe make be clearer to say "which uses this mechanism", dropping the "also"? Or maybe "which uses this function"? Or am I just being pedantic?

Reply:
inclined to say the latter :) but if we're being pedantic, why not `(which also uses make-options-doc)`?

Reply:
> but if we're being pedantic, why not `(which also uses make-options-doc)`?

My issue with that is that this *is* `make-options-doc`, so it seems kind of redundant? Not sure, why do you think that's better? Since it's more explicit?

Reply:
pretty much, yeah. the nixpkgs docs don't really need the merging aspect iwrc, it's just something they get for free. either phrasing is totally fine though, you decide which you like more :)

Reply:
> most people on Darwin will just be hacking on the Nixpkgs manual

not sure what that means. Is that not true for people on Linux?

Reply:
The NixOS manual also exists :-)